android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/BaseDDLFieldTextView.java

```diff
@@ -108,6 +109,11 @@ public void setupFieldLayout() {
         if (this.field.getPlaceHolder() != null && !this.field.getPlaceHolder().isEmpty()) {
             textEditText.setHint(this.field.getPlaceHolder());
         }
+
+        if (this.field.getTip() != null && !this.field.getTip().isEmpty()) {
```

Comment:
This seems like a perfect case for a kotlin extension, don't you think? this.field.getTip().isNullOrEmpty() :)
(just and idea)

Reply:
if tip is String Kotlin extensions already provide it @mclomello 
```
var strNullable : String?
strNullable = "adfasdf"
strNullable.isNullOrEmpty() //This is valid
```

Reply:
Yeah, you are right, I thought it was kotlin 😂